src/main/webapp/app/orion/assessment/orion-assessment.service.ts

```diff
@@ -69,6 +69,7 @@ export class OrionAssessmentService {
         const exportOptions: RepositoryExportOptions = {
             exportAllParticipants: false,
             filterLateSubmissions: false,
+            excludePracticeSubmissions: false,
```

Comment:
I think it's a bit weird to have this option appear in the `orion-assessment` service. Also, I'm not entirely sure whether Orion needs the practice submissions 🤔

Reply:
I think it is fine since it conforms to `RepositoryExportOptions`. Do you think that is okay?

Reply:
I think so, yes 👍